tests/tests_api/test_get_main_page.py

```diff
@@ -0,0 +1,12 @@
+import allure
+
+from api_client.methods_api import ApiClient
+from core.config import Configuration
+
+
+class TestGetMainPage:
+    @allure.story('Get Mainpage')
+    def test_get_mainpage(self):
+        api_page = ApiClient()
+        response = str(api_page.get(Configuration.BASE_URL))
```

Comment:
The Response object has a lot of useful info like response.status_code https://docs.python-requests.org/en/master/
Use built in status code instead of converting the response to str

Reply:
Done.